### ISSUE TITLE
rendition.location.atEnd not work

### ISSUE THREAD

**Issue by dnzl123** [futurepress/epub.js#975]
i use rendition.next(); or  const cfi=this.book.locations.cfiFromPercentage(1); this.rendition.display(cfi);to the end of the epub book.but, rendition.location.atEnd return undefind. and rendition.location.displayed.page; rendition.location.displayed.total; are not equal.

i use this in android webview;

**Comment by dnzl123** [comment]
this issue happened in some android mobile phone. once it happened, every section's last page i use  console.log(rendition.location.displayed.page); console.log(rendition.location.displayed.total); total is one more than page.

**Comment by yh54321** [comment]
@dnzl123 Yes I can confirm this. The issue seems to arise when flipping from the first page of a section to the second page. While the page flip is successful and the CFIs of the location change, the page number in the location does not advance by 1. As a result, when the the reader flips all the way to the last page of the section, the page recorded in the location will be 1 less than the total, and thus the 'atEnd' flag is not true.

I have not yet found a solution to this and it is not yet clear to me what causes this to occur.
